README.md

```diff
@@ -172,7 +173,46 @@ It returns a function that accepts args, an object that contains the parameters
 If you are looking for a breakdown of the API changes from AWS SDK for JavaScript v2 to v3,
 we have them listed in [UPGRADING.md](https://github.com/aws/aws-sdk-js-v3/blob/main/UPGRADING.md).
 
-### Install from Source
+## Working with the SDK in Lambda
+
+### General Info
+The embedded JS SDK v3 version that is vended by lambda is set to a specific minor version, and NOT the latest version. To check the minor version used by Lambda please refer to [Lambda rutimes doc page](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html).
```

Comment:
"embedded" is this the official term? use "Lambda provided AWS SDK"

use full names of products:
lambda -> AWS Lambda
JS SDK v3 -> AWS SDK for JavaScript (v3)